<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fulltext_search</class>
 <widget class="QWidget" name="fulltext_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="titleList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="showFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>951</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
    </font>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>27</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>TITLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text Searchï¼š</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="searchText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>253</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="characters">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Number of characters: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="words">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Number of words: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sentences">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Number of sentences:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
